pegasus/sites.v3/code.org/public/js/volunteer-unsubscribe.js

```diff
@@ -0,0 +1,46 @@
+function processResponse(data)
+{
+  $('#unsubscribe-volunteer-form').hide();
+  $('#thanks').show();
+}
+
+function processError(data)
+{
+  $('#error-message').text('An error occurred. Please try again or contact us if you continue to receive this error.').show();
+  $('body').scrollTop(0);
+  $("#btn-submit").removeAttr('disabled');
+  $("#btn-submit").removeClass("button_disabled").addClass("button_enabled");
+}
+
+function unsubscribeVolunteerList()
+{
+  $("#btn-submit").attr('disabled','disabled');
+  $("#btn-submit").removeClass("button_enabled").addClass("button_disabled");
+
+  secret = $('#volunteer-secret').text()
+  form_results = $('#unsubscribe-volunteer-form').serializeArray();
+
+  var data = {};
+  $(form_results).each(function(index, obj){
```

Comment:
Identifier 'form_results' is not in camel case.<br>'form_results' is not defined.